test/test_annotatable.cpp

```diff
@@ -1,7 +1,8 @@
-#include <sstream>
-#include <cstdio>
 #include "tree-annotatable.hpp"
-#include "assert.hpp"
+
+#include <gmock/gmock.h>
```

Comment:
Is there not just a `gtest/gtest.h` header?

Reply:
Thanks for the comment, Pablo! I've just removed the `gmock` dependency. This is one of the dangers of the copy-and-paste programming! hehe.